basics/indexing/star-tree-index.md

```diff
@@ -1,16 +1,22 @@
+---
+description: Learn what a star-tree index is and the concept of how one works.
+---
+
 # Star-Tree Index
 
+In this page you will learn what a star-tree index is and gain a conceptual understanding of how one works.
+
 Unlike other index techniques which work on a single column, the star-tree index is built on multiple columns and utilizes pre-aggregated results to significantly reduce the number of values to be processed, resulting in improved query performance.
 
 One of the biggest challenges in realtime OLAP systems is achieving and maintaining tight SLAs on latency and throughput on large data sets. Existing techniques such as [sorted index](forward-index.md) or [inverted index](inverted-index.md) help improve query latencies, but speed-ups are still limited by the number of documents that need to be processed to compute results. On the other hand, pre-aggregating the results ensures a constant upper bound on query latencies, but can lead to storage space explosion.
 
-Here we introduce the **star-tree** index to utilize the pre-aggregated documents in a smart way to achieve low query latencies but also use the storage space efficiently for aggregation/group-by queries.
+Here we introduce the **star-tree** index to utilize the pre-aggregated documents in a way that achieves low query latencies but also uses the storage space efficiently for aggregation and group-by queries.
 
 {% embed url="https://www.youtube.com/watch?v=bwO0HSXguFA" %}
 
-### Existing solutions
+## Existing solutions
 
-Consider the following data set as an example to discuss the existing approaches:
+Consider the following data set, which is used here as an example to discuss the existing approaches:
```

Comment:
Would it make sense to say "....as an example to discuss each index type as a solution?

Also, what are these indexes a solution for?

Reply:
I wouldn't use "solution" here. I do think it's reasonable to change this to "...to discuss these index types."

unless you go really high level (they make queries faster), the various indices aren't solving the same problem. while that makes me not like "approaches" it also makes me not like "solutions" here.

Reply:
```suggestion
Consider the following data set, which is used here as an example to discuss these index types:
```